frontend/src/store/index.js

```diff
@@ -15,7 +15,7 @@ import appFiles from "./Files/index";
 import appGit from "./Git/index";
 
 export default store(function () {
-	const Store = createStore({
```

Comment:
There is a typo in the commit "Update sotre function" -> "Update s**to**re function"